packages/components/src/query-controls/types.ts

```diff
@@ -107,7 +107,8 @@ type BaseQueryControlsProps = {
 	 * Start opting into the larger default height that will become the
 	 * default size in a future version.
```

Comment:
Should we remove the old docs as they're no longer relevant?

```suggestion
```

Reply:
I'm hoping it will be helpful to extenders, at least for a while, until this deprecation propagates. They will see the prop marked as deprecated in their IDEs, and they might want to know what that prop was about.